mqtt/mqtt.js

```diff
@@ -54,7 +54,7 @@ var logger = bunyan.createLogger({name: "mqtt", pinoOptions: {level: 30}}),
 logging({
     instance: aedes,
     servers: servers,
-    pinoOptions: {level: config.log_level}
+    pinoOptions: {level: 30}
```

Comment:
Hard-coding?

Reply:
Yes, this is not our logger :)

Reply:
Yes, this is not our logger, and doesn't affect logs that we control.